menu.js

```diff
@@ -20,6 +20,10 @@ module.exports = function createMenu ({ createWindow, updatePlugins }) {
           click (item, focusedWindow) {
             if (focusedWindow) {
               focusedWindow.rpc.emit('preferences');
+              return;
+            } else if (!mainWindow.isFocused()) {
+              mainWindow.on('focus', () => mainWindow.rpc.emit('preferences'));
```

Comment:
- What if there's no `mainWindow` (all closed)? 
- If you don't use `once`, wouldn't preferences fire every subsequent time the window gets `focus`?